ci/build_macos.sh

```
@@ -0,0 +1,58 @@
+#!/bin/bash -e
+# Generated with Claude Code (https://claude.com/claude-code)
+
+CI_DIR=$(readlink -f $(dirname $0))
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)